app/integrations/zendesk/scheduler.py

```
@@ -955,12 +959,14 @@ async def _zendesk_run_internal_retrieval_preflight(
 
         supa = get_supabase_client()
         resp = await supa._exec(
-            lambda: supa.client.table("feedme_text_chunks")
-            .select("conversation_id,chunk_index,content")
-            .eq("conversation_id", conv_int)
-            .in_("chunk_index", wanted_indices)
-            .order("chunk_index")
-            .execute()
+            lambda: (
+                supa.client.table("feedme_text_chunks")
```

Ruff (0.14.14)

[warning] Function definition does not bind loop variable `supa`

(B023)